### ISSUE TITLE
Upgrade/Downgrade

### ISSUE THREAD

**Issue by Pomdre** [blesta/module-pterodactyl#40]
When using the upgrade/downgrade check boxes of the use module, the module does not update the server ram. 
![image](https://user-images.githubusercontent.com/30056774/80220061-98a36f80-8643-11ea-9d7c-850a7cc30539.png)

**Comment by Pomdre** [comment]
This should be possible to fix I think.
https://dashflo.net/docs/api/pterodactyl/v0.7/#update-server-build-configuration

**Comment by kakduman** [comment]
This is still possible after updating to v1: https://dashflo.net/docs/api/pterodactyl/v1/#req_11fc764c3ed648ca8e6d60bff860ca6d

It is by far the largest issue facing Blesta and the primary reason why I'm recommending WHMCS over Blesta to people who ask me right now. The inability to upgrade/downgrade servers, even after 10 months of an issue being open to add the feature, practically makes Blesta's Pterodactyl interface obsolete.

I tried looking at the source and was unable to figure it out due to the language. I created a python script to upgrade/downgrade servers, and I would be happy to send that if it helped get this feature implemented. Pterodactyl's WHMCS module is open-source and also supports upgrade/downgrades, if looking at that could help Blesta implement it: https://github.com/pterodactyl/whmcs/blob/master/pterodactyl/pterodactyl.php

This is also already an issue at https://dev.blesta.com/browse/CORE-4123, but no progress seems to have been made since the issue was created.

Edit: it looks like the link to the Pterodactyl documentation above does not go to the correct place. It'd use the /api/application/servers/{server}/build method, so opening that method after going to the documentation should provide a comprehensive overview of how to use it.
